app/models/event/plan/argosy_2024.rb

```diff
@@ -31,6 +31,10 @@ def features
         super - %w[promotions google_workspace]
       end
 
+      def unrestricted_disbursements_allowed?
+        false
+      end
+
```

Comment:
You can inherit from standard - we don't need to explicitly define it on every plan